schemas/2015-10-01/Microsoft.Media.json

```diff
@@ -0,0 +1,94 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-10-01/microsoft.media.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Media",
+  "description": "Microsoft Media Service Resource Type",
+  "resourceDefinitions": {
+    "mediaServices": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "Microsoft.Media/mediaServices"
+          ]
+        },
+        "apiVersion": {
+          "enum": [
+            "2015-10-01"
+          ]
+        },
+        "name": {
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern": "^[a-z0-9]",
+              "minLength": 3,
+              "maxLength": 26
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        },
+        "properties": {
+        "oneOf": [
+          {
+            "$ref": "#/definitions/MediaServiceProperties"
+          },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        }
+      },
+      "required": [
+        "type",
+        "apiVersion",
+        "properties",
+        "location"
+      ],
+      "description": "Microsoft.Media/mediaServices"
+    }
+  },
+  "definitions" : {
+    "MediaServiceProperties": {
+      "properties": {
+        "apiEndpoints": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiEndpoint"
+          }
+        },
+        "storageAccounts": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StorageAccount"
+          }
+        }
+      },
+      "type": "object"
+    },
+    "ApiEndpoint": {
+      "properties": {
+        "endpoint": {

```

Comment:
Api endpoints are returned for a create.

From: Wenjun Wu [mailto:notifications@github.com]
Sent: Monday, June 20, 2016 4:17 PM
To: Azure/azure-resource-manager-schemas <azure-resource-manager-schemas@noreply.github.com>
Cc: Brian Blum <brianbl@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [Azure/azure-resource-manager-schemas] Add Microsoft.Media 2015-10-01 schema (#185)


In schemas/2015-10-01/Microsoft.Media.json<https://github.com/Azure/azure-resource-manager-schemas/pull/185#discussion_r67784117>:

> +          "items": {

> +            "$ref": "#/definitions/ApiEndpoint"

> +          }

> +        },

> +        "storageAccounts": {

> +          "type": "array",

> +          "items": {

> +            "$ref": "#/definitions/StorageAccount"

> +          }

> +        }

> +      },

> +      "type": "object"

> +    },

> +    "ApiEndpoint": {

> +      "properties": {

> +        "endpoint": {

any of endpoint or majorVersion required for ApiEndpoint?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Azure/azure-resource-manager-schemas/pull/185/files/77daeed5044ea58e8d5a2e7dddbef1bb6e8d2f7e#r67784117>, or mute the thread<https://github.com/notifications/unsubscribe/AEH9bGiTg1F9eI6yMVzYIsqVDOnqt9jYks5qNx9ggaJpZM4I6GNO>.